src/context-menu-builder.js

```diff
@@ -234,41 +234,42 @@ export default class ContextMenuBuilder {
 
     // Ensure that we have valid corrections for that word
     let corrections = await this.spellCheckHandler.getCorrectionsForMisspelling(menuInfo.misspelledWord);
-    if (!corrections || !corrections.length) {
-      return menu;
-    }
-
-    corrections.forEach((correction) => {
-      let item = new MenuItem({
-        label: correction,
-        click: () => target.replaceMisspelling(correction)
+    if (corrections && corrections.length) {
+      corrections.forEach((correction) => {
+        let item = new MenuItem({
+          label: correction,
+          click: () => target.replaceMisspelling(correction)
+        });
+  
+        menu.append(item);
       });
-
-      menu.append(item);
-    });
-
-    this.addSeparator(menu);
-
-    // Gate learning words based on OS support. At some point we can manage a
-    // custom dictionary for Hunspell, but today is not that day
-    if (process.platform === 'darwin') {
-      let learnWord = new MenuItem({
-        label: this.stringTable.addToDictionary(),
-        click: async () => {
-          // NB: This is a gross fix to invalidate the spelling underline,
-          // refer to https://github.com/tinyspeck/slack-winssb/issues/354
-          target.replaceMisspelling(menuInfo.selection);
```

Comment:
`selection` property doesn't exist on `menuInfo`. The added code below fixes this and also allows for Windows users to save words through `winUserWords`